lib/change_stream.js

```diff
@@ -368,6 +368,15 @@ function processNewChange(args) {
   const change = args.change;
   const callback = args.callback;
   const eventEmitter = args.eventEmitter || false;
+
+  if (changeStream.isClosed()) {
+    if (eventEmitter) return;
+
+    const error = new Error('ChangeStream is closed');
```

Comment:
I think this should be a `MongoError` right?

Reply:
also nit, but should `ChangeStream` just be `change stream`? It makes it look like a product name here 😄

Reply:
Was referring to the `class ChangeStream` here. Figured it was better to specify the constructor object.

Also, I specifically resisted throwing a `MongoError` b/c right now we use `MongoError` mostly for server-side errors. We don't really have a concept of a `MongoDriverError`.

Reply:
:+1: on point one
For the second point, I still think it makes sense to mark it as a `MongoError` until such an error class exists. It makes it harder to catch errors generically limited to the driver if we also sometimes throw plain `Error` s, at very least even if/when a `MongoDriverError` is introduced it will be a subclass of `MongoError`